app/uk/gov/hmrc/fileupload/frontendGlobal.scala

```diff
@@ -125,8 +125,8 @@ class ApplicationModule(context: Context) extends BuiltInComponentsFromContext(c
 
   // quarantine
   lazy val quarantineRepository = quarantine.Repository(db)
-  lazy val retrieveFile = quarantineRepository.retrieveFile _
-  lazy val getFileFromQuarantine = QuarantineService.getFileFromQuarantine(retrieveFile) _
+  lazy val retrieveFileFromQuarantineBucket = s3Service.retrieveFileFromQuarantine _
+  lazy val getFileFromQuarantine = QuarantineService.getFileFromQuarantine(retrieveFileFromQuarantineBucket) _
```

Comment:
Here you should be able to pass `s3Service.retrieveFileFromQuarantine` directly to `getFileFromQuarantine` without a need for intermediate variable `retrieveFileFromQuarantineBucket`